butterfree/migrations/migrator.py

```diff
@@ -0,0 +1,41 @@
+"""Holds the Migrator Class."""
+
+from typing import Callable, List, Tuple
+
+from butterfree.pipelines import FeatureSetPipeline
+from butterfree.transform import FeatureSet
+
+
+class Migrator:
+    """Execute migration operations in a Database based on pipeline Writer.
+
+    Attributes:
+        pipelines: list of Feature Set Pipelines to use to migration.
+
+    """
+
+    def __init__(self, pipelines: List[FeatureSetPipeline]) -> None:
+        self.pipelines = pipelines
+
+    def _parse_feature_set_pipeline(
+        self, pipeline: FeatureSetPipeline
+    ) -> List[Tuple[Callable, FeatureSet]]:
+        feature_set = pipeline.feature_set
+        migrations = [
+            writer.db_config._migration_class for writer in pipeline.sink.writers
```

Comment:
Here, we would have a new parameter within ```db_config``` that would return, for instance, a ```CassandraMigration()``` as a result? Wouldn't it be possible to establish this same logic by parsing the writers from the feature pipeline and, for each writer, we could retrieve the corresponding migration class?

Reply:
How are we going to make sure that every Historical writer is Metastore or every Online writer is Cassandra? So I think the best way to know where to apply the migration is in db_config.